### ISSUE TITLE
Figure out how to handle PDS4 versions

### ISSUE THREAD

**Issue by rfrenchseti** [SETI/rms-pdsfile#31]
Pds4File currently does not have a mechanism for handling versions of PDS4 bundles. We need to figure out how versioning is going to work (including whether files are copied or just symlinked), and then figure out how the rules in Pds4File should be constructed.

**Comment by matthewtiscareno** [comment]
@markshowalter's memo ([attached](https://github.com/SETI/rms-pdsfile/files/14891484/PDS4-VERSIONING.txt), and also in Dropbox under RMS Policy Documents) is the canonical way to go.  However, since we don't have software tools in place to manage such a complex system, we are currently simply doing what we did for PDS3.

**Comment by rfrenchseti** [comment]
Looking at the PdsFile code, it may just do the right thing with versions if we follow the same naming standard we used for PDS3. For `uranus_occs_earthbased`, we should not include the duplicated bundle, because it will show up all the time (versioning happens during download, not during search). Also we'll have to add a rule for the user's guide with the alternate naming.

**Comment by matthewtiscareno** [comment]
Sounds good.  So do I have any further action items regarding this issue?

**Comment by rfrenchseti** [comment]
I guess one open question is whether you are versioning the bundles or the bundleset. Does everything go under `uranus_occs_earthbased` and there are now twice as many bundles? Or are there two bundlesets `uranus_occs_earthbased` and `uranus_occs_earthbased_v1`?

**Comment by matthewtiscareno** [comment]
Looking at Viewmaster, it looks like superseded volumesets reside in directories containing the version numbers, but their volume subdirectories do not.  Perhaps do the same?

**Comment by rfrenchseti** [comment]
I think that would be cleaner as long as it meets whatever PDS4 standards we need to meet.

Do you have to change any LIDVIDs within the bundles?

**Comment by matthewtiscareno** [comment]
Part of creating a superseding bundle is fixing the LIDVIDs for the new version.  We wouldn't update LIDVIDs in an old bundle.  It is preserved as a product of its time.

**Comment by rfrenchseti** [comment]
Right. I just wanted to make sure that was being done for the new `uranus_occs_earthbased` bundle that is about to be created.

**Comment by matthewtiscareno** [comment]
I believe @mace-space is on the case with regard to that.

**Comment by matthewtiscareno** [comment]
(alliteration not intended)  :)

**Comment by mace-space** [comment]
Trying to keep pace

**Comment by rfrenchseti** [comment]
It has now been decided that there is a `superseded` directory under the bundleset directory, like `bundles/uranus_occs_earthbased/superseded`. This directory should be ignored when enumerating bundles, but should be searched when looking for OPUS download products.
